runmariadb

```diff
@@ -2,8 +2,19 @@
 
 set -eu
 
+AUTHROOT=/auth/mariadb
+
+if [[ -f "${AUTHROOT}/username" ]]; then
```

Comment:
This is now overriding password from env variable. Env variables should have higher priority than files. Check should be added that if MARIADB_USER is empty, and if this file exists, then it is read.